deploy/servicemesh-operator.yaml

```diff
@@ -198,6 +198,18 @@ rules:
   - jaegers
   verbs:
   - '*'
+- apiGroups:
+  - kiali.io
+  resources:
+  - kialis
+  verbs:
+  - '*'
+- apiGroups:
+  - monitoring.kiali.io
```

Comment:
I'm not sure if istio-operator even needs this monitoring.kiali.io apiGroup perms.. I'll test that soon. I think this can be removed.